{{cookiecutter.project_repo}}/CONTRIBUTING.rst

```diff
@@ -59,10 +59,10 @@ Get Started!
 
 Ready to contribute? Here's how to set up `{{ cookiecutter.project_slug }}` for local development.
 
-1. Fork the `{{ cookiecutter.project_slug }}` repo on GitHub.
+1. Fork the `{{ cookiecutter.project_repo }}` repo on GitHub.
```

Comment:
Actually.. you are asking for people to fork aren't you.

That might be how you keep the liccheck up-to-date.